test/functional/test_site/testAntiFOUCStyles.md

```diff
@@ -0,0 +1,20 @@
+<!-- Anti-FOUC styles should be applied to navbar. -->
+
+<navbar placement="top" type="inverse">
+  <li>One</li>
+  <li>Two</li>
+</navbar>
+
+<!-- Anti-FOUC styles should be applied to dropdown. -->
+
+<dropdown text="test" class="test-class">
+  <li>One</li>
+  <li>Two</li>
+</dropdown>
```

Comment:
Both these examples are in a very short page content, so upon clicking the dropdowns, I did not realise I had to scroll to see the list items. 

Maybe add some filler text / headings to label what this dropdown is testing? (Also it will increase the height of the page content, removing the need to scroll)